libpages/server.go

```diff
@@ -298,8 +298,8 @@ func ListenAndServe(ctx context.Context,
 		HostPolicy: server.allowConnectionTo,
 	}
 
-	if config.UseDiskCacheForDev {
-		manager.Cache = autocert.DirCache("./kbp-cert-cache-dev")
+	if config.UseDiskCertCache {
+		manager.Cache = autocert.DirCache("./kbp-cert-cache")
```

Comment:
Should this use a different directory if `!fProd`, so the certs don't get mixed up if you switch between modes on the same instance?

Reply:
I figured we'd almost never use the same instance for testing and prod, so I just removed the `-dev` suffix. But having different suffix sounds good with me too; will change!